receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper.go

```diff
@@ -132,6 +134,10 @@ func (s *scraper) scrape(_ context.Context) (pmetric.Metrics, error) {
 			errs.AddPartial(fileDescriptorMetricsLen, fmt.Errorf("error reading open file descriptor count for process %q (pid %v): %w", md.executable.name, md.pid, err))
 		}
 
+		if err = s.scrapeAndAppendSignalsPendingMetric(now, md.handle); err != nil {
+			errs.AddPartial(diskMetricsLen, fmt.Errorf("error reading pending signals for process %q (pid %v): %w", md.executable.name, md.pid, err))
```

Comment:
```suggestion
			errs.AddPartial(signalMetricsLen, fmt.Errorf("error reading pending signals for process %q (pid %v): %w", md.executable.name, md.pid, err))
```

Reply:
Oops I guess that's rebase magic doing its work. Thanks for noticing this Evan!